src/main/java/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -717,7 +717,7 @@ private ListenableFuture<Boolean> initializeDraft() {
     DcMsg draft = dcContext.getDraft(chatId);
     final String sharedText = RelayUtil.getSharedText(this);
 
-    if (draft == null) {
+    if (!draft.isOk()) {
```

Comment:
wouldn't this cause a crash if draft is null??? I think what you wanted is:
```suggestion
    if (draft == null || !draft.isOk()) {
```

Reply:
wow so this was never entering that path